spec/models/ride_spec.rb

```diff
@@ -3,5 +3,6 @@
 RSpec.describe Ride, type: :model do
   describe "relationships" do
     it { should belong_to(:amusement_park) }
+    it { should have_many(:mechanics).through(:ride_mechanics) }
```

Comment:
It's also best practice to include `it { should have_many(:ride_mechanics } ` as well (i.e. one test for each association